README.md

```
@@ -1,214 +1,761 @@
+### Motivation
+While `orpc` is great and you can use it for direct function calls (and not network requests with `createRouterClient`), and for example for building "type-safe SDK"s, it does have a few opinions that may get in the way. I use it extensively in my projects, but `zagora` is smaller and even more focused approach - i always wanted "just functions" where you define input, outputs, and you get error-safe, typed function back not a wrapper around it.
+
+Both `tRPC` and `oRPC` are promoted as "backend", or specifically for when you're building "apps". Recently, all major frameworks also introduced similar concepts, like "server actions" and so on. All that is cool, but `zagora` is focused on building just functions, a low-level library for building other libraries - I have a lot of them, so i need a simple way for building type-safe and error-safe functions, where i don't necessarily need network layer and i don't need "routers" concept, and etc.
+
+They are built around the network, Zagora is built around functions with excellent egonomics, developer experience, and no assumptions. **It produces just functions, i cannot stress that enough.**
+
+### Why Zagora over oRPC/tRPC/neverthrow/effect.ts?
+
+- Zagora is focused on producing "just functions", not networks, routers, or groups.
+- oRPC and tRPC does not "support" creating synchornous functions, they always are async
+  + in contrast, Zagora does not use `async/await `anywhere in the codebase, but `instanceof Promise` checks
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)